src/services/api.js

```diff
@@ -1,7 +1,15 @@
+const urlBase = 'https://api.mercadolibre.com/sites/MLB/search?';
+
 export async function getCategories() {
   // Implemente aqui
+  const response = await fetch('https://api.mercadolibre.com/sites/MLB/categories');
+  const categoriesList = await response.json();
+  return categoriesList;
 }
 
-export async function getProductsFromCategoryAndQuery(/* categoryId, query */) {
+export async function getProductsFromCategoryAndQuery(categoryId, query) {
   // Implemente aqui! Quando o fizer, descomente os parâmetros que essa função recebe
+  const response = await fetch(`${urlBase}${categoryId}${query}`);
```

Comment:
@guihtryb @LeonardoZaramello O que vocês acham de adicionar um '&' entre a categoria e a query, e colocar 'category=' e 'q=' antes de cada literal? :smile: :thinking: 
```suggestion
  const response = await fetch(`${urlBase}category=${categoryId}&q=${query}`);
```